colossus-metrics/src/test/scala/colossus/metrics/CollectionSpec.scala

```diff
@@ -9,14 +9,14 @@ class CollectionSpec extends WordSpec with MustMatchers{
 
     val collection = Collection.withReferenceConf(Seq(1.minute, 1.second))
 
-    implicit val namespace = MetricContext("/", collection)
+    implicit val namespace: MetricContext = MetricContext("/", collection)
 
     "getOrAdd should add a new Collector" in {
       //note, this constructor implicitly adds its constructed object into this collection
       val foo = MetricAddress("foo")
       val rate = Rate(foo, false, true)
 
-      val x = namespace.collection.collectors.get(foo)
+      val x = namespace.collection.get(foo).get
```

Comment:
For this,and other instances of where you are calling `get` directly, ScalaTest does have at `OptionValues` trait which provides a  `value` function. This behaves like `get' but actually provides a more appropriate error message if you are `get`ing a None (vs NPE)